incubator/postgres-operator/0.0.1/Chart.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: postgres-operator
+version: 0.0.1
+upstream_version: 1.6.2
+appVersion: "auto"
+description: Postgres Operator creates and manages PostgreSQL clusters running in Kubernetes
+type: application
+deprecated: false
+home: https://github.com/truecharts/apps/tree/master/incubator/postgres-operator
+icon: https://truecharts.org/_static/img/postgres-operator-icon.png
+keywords:
+  - database
+  - HA
+  - postgresql
+  - SQL
+  - postgres
+  - operator
+  - cloud-native
+  - patroni
+  - spilo
+sources:
+  - https://github.com/zalando/postgres-operator
+dependencies:
+  - name: common
+    repository: https://truecharts.org/
+    version: 2.0.2
+    # condition:
+  - name: postgres-operator-ui
+    repository: https://raw.githubusercontent.com/zalando/postgres-operator/master/charts/postgres-operator-ui/
+    version: 1.6.2
+    # condition:
+  - name: postgres-operator
+    repository: https://raw.githubusercontent.com/zalando/postgres-operator/master/charts/postgres-operator/
+    version: 1.6.2
+    # condition:
```

Comment:
this will fail for the second installed instance - are you working in a conditional add?

Reply:
oh, I see - the operator is installed once while postgres-operator-manifests is installed once per DB so this should work fine.

Reply:
Yes it is :)